front/src/config/i18n/fr.json

```diff
@@ -1076,12 +1076,18 @@
       "turnOffLights": {
         "label": "Sélectionnez les lumières que vous souhaitez éteindre"
       },
+      "toggleLights": {
+        "label": "Sélectionnez les lumière que vous souhaitez inverser"
```

Comment:
*lumières

Reply:
it is true that in french it is not so pretty...

What about this?
![image](https://user-images.githubusercontent.com/5940493/208499218-ec15dac5-55f7-4430-b10a-de2bdf4fa3e0.png)

But OK for a more readable little text

Reply:
It's a bit dirty to have this long text in bold, what about having a small text in bold and an explanation in small ?

Reply:
I keep the same layout as in other actions. Should not be part of a new PR?

Reply:
Yeah but other actions are not that long, so it's a problem for this one! 

Can be a conditional translations if you still want to re-use the same components